hooks/web-socket/constants.ts

```diff
@@ -0,0 +1,145 @@
+
+    },
+    [CurrentPage.ZERO_SHOT]: {
+        [CurrentPageSubKey.NONE]: ['labeling_task_updated', 'labeling_task_created', 'label_created', 'label_deleted', 'labeling_task_deleted', 'information_source_deleted', 'information_source_updated', 'payload_update_statistics', 'payload_finished', 'payload_failed', 'payload_created', 'zero-shot', 'zero_shot_download']
+    },
+    [CurrentPage.LABELING_FUNCTION]: {
+        [CurrentPageSubKey.NONE]: ['labeling_task_updated', 'labeling_task_created', 'label_created', 'label_deleted', 'labeling_task_deleted', 'information_source_deleted', 'information_source_updated', 'model_callback_update_statistics', 'payload_progress', 'payload_finished', 'payload_failed', 'payload_created', 'payload_update_statistics'],
+    },
+    [CurrentPage.CROWD_LABELER]: {
+        [CurrentPageSubKey.NONE]: ['labeling_task_updated', 'labeling_task_created', 'label_created', 'label_deleted', 'labeling_task_deleted', 'information_source_deleted', 'information_source_updated', 'model_callback_update_statistics']
+    },
+    [CurrentPage.ACTIVE_LEARNING]: {
+        [CurrentPageSubKey.NONE]: ['labeling_task_updated', 'labeling_task_created', 'label_created', 'label_deleted', 'labeling_task_deleted', 'information_source_deleted', 'information_source_updated', 'model_callback_update_statistics', 'embedding_deleted', 'embedding', 'payload_finished', 'payload_failed', 'payload_created', 'payload_update_statistics']
+    },
+    [CurrentPage.HEURISTICS]: {
+        [CurrentPageSubKey.NONE]: ['labeling_task_updated', 'labeling_task_created', 'labeling_task_deleted', 'information_source_created', 'information_source_updated', 'information_source_deleted', 'payload_finished', 'payload_failed', 'payload_created', 'payload_update_statistics', 'embedding_deleted'],
+
+    },
+    [CurrentPage.EDIT_RECORDS]: {
+        [CurrentPageSubKey.NONE]: ['calculate_attribute']
+    },
+    [CurrentPage.DATA_BROWSER]: {
+        [CurrentPageSubKey.NONE]: ['data_slice_created', 'data_slice_updated', 'data_slice_deleted', 'label_created', 'label_deleted', 'labeling_task_deleted', 'labeling_task_updated', 'labeling_task_created', 'information_source_created', 'information_source_updated', 'information_source_deleted', 'attributes_updated', 'calculate_attribute', 'embedding', 'embedding_deleted'],
+
+    },
+    [CurrentPage.ATTRIBUTE_CALCULATION]: {
+        [CurrentPageSubKey.NONE]: ['attributes_updated', 'calculate_attribute', 'tokenization', 'knowledge_base_updated', 'knowledge_base_deleted', 'knowledge_base_created'],
+
+    },
+    [CurrentPage.ADMIN_PAGE]: {
+        [CurrentPageSubKey.NONE]: ['pat']
+    },
+    [CurrentPage.NOTIFICATION_CENTER]: {
+        [CurrentPageSubKey.NONE]: ['notification_created', 'project_deleted', 'config_updated', 'admin_message'],
+
+    }
+}
+
+export const WHITELIST_LOOKUP_COGNITION = {
+    [CurrentPage.PROJECTS]: {
+        [CurrentPageSubKey.NONE]: ['admin_message']
+    }
+}
```

Comment:
To my understanding the CurrentPage is basically an indicator for the whitelists & where it's used. It doesn't have to be a physical page URL just a grouping. 

Since admin messages are not "Page" related but more global I'd assume a more global key like CurrentPage.COGNITION_LAYOUT or something like that